File: main/src/main/java/net/fwitz/math/main/binary/complex/functions/trig/inverse/AsinZ.java
package net.fwitz.math.main.binary.complex.functions.trig.inverse;

import net.fwitz.math.binary.complex.Complex;
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot;

public class AsinZ {
    public static void main(String[] args) {
        new ComplexFunctionPlot("asin(z)")
                .computeFn(Complex::asin)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/complex/functions/trig/inverse/AsinZ.kt
package net.fwitz.math.main.binary.complex.functions.trig.inverse

import net.fwitz.math.binary.complex.Complex
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot

object AsinZ {
    @JvmStatic
    fun main(args: Array<String>) = ComplexFunctionPlot("asin(z)")
        .computeFn(Complex::asin)
        .render()
}